code/src/shared/modules/application/application.tsx

```diff
@@ -4,6 +4,7 @@ import AppBanner from "@/layouts/app-banner/AppBanner";
 import AppFooter from "@/layouts/app-footer/AppFooter";
 import "@/shared/modules/application/application.scss";
 import PageWrapper from "@/layouts/app-wrapper/PageWrapper";
+import AppProductsList from "@/layouts/app-products/AppProductsList";
```

Comment:
Organize imports (you can find an example in our [PR](https://github.com/idx-academy/spa-orders/pull/32)).

Reply:
Fixed